packages/tests/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Tests setup for /id projects",
   "main": "index.js",
   "scripts": {
-    "test:e2e": "playwright test"
+    "test:e2e": "playwright test --debug"
```

Comment:
I guess we shouldn't commit this?

Reply:
pretty sure you caught an older version, I removed it 😓